Spokestack/SpeechPipeline.swift

```diff
@@ -186,10 +186,19 @@ import Dispatch
     /// - SeeAlso: `SpeechConfiguration`
     /// - Parameters:
     ///   - key: Configuration property name
-    ///   - value: Configuration property name
+    ///   - value: Configuration property value
+    /// - Note: "tracing" key must have a value of `Trace.Level`, eg `Trace.Level.DEBUG`.
```

Comment:
Doesn't this change make for an otherwise unnecessary backflip during RN setup?

Reply:
This change has no effect on the current RN setup.